client/styles/index.scss

```diff
@@ -3,6 +3,7 @@
 @import './events-list.scss';
 @import './add-event-container.scss';
 @import './add-event-form.scss';
+@import './react-geosuggest.scss';
```

Comment:
this is overrides for our implementation (this css in react-geosuggest/module is only for the example for that component).